lms/djangoapps/support/static/support/jsx/entitlements/components/Main/MainContainer.jsx

```diff
@@ -0,0 +1,24 @@
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import * as actionCreators from '../../data/actions/actionCreators';
+import Main from './Main';
+
+function mapStateToProps(state) {
+	return {
+		entitlements: state.entitlements,
```

Comment:
nit: Are these tabs or spaces?  Also what is the code style for spacing in jsx files?

Spacing across all the jsx files seems to be quite large. If it is the style just let me know.